frontend/www/js/omegaup/components/course/DescriptionCard.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <div class="card-header mb-3">
+    <h1>{{ T.courseCardAboutCourses }}</h1>
+    <p v-html="T.courseCardDescriptionCourses"></p>
+    <div class="text-right align-middle">
+      <a v-bind:href="`https://blog.omegaup.com/cursos-en-omegaup/`">{{
```

Comment:
```suggestion
      <a href="https://blog.omegaup.com/cursos-en-omegaup/">{{
```

`v-bind` sólo se necesita usar en caso de que existan variables. para una constante, se puede omitir.